### ISSUE TITLE
download - no more

### ISSUE THREAD

**Issue by illtellyoulater** [darrinhenein/VerticalTabs#26]
the download link is broken

**Comment by ignlg** [comment]
Get it here: https://github.com/darrinhenein/VerticalTabs/raw/master/VerticalTabs.xpi

**Comment by illtellyoulater** [comment]
I know, I just wanted to inform you that the link was not working.

**Comment by tobia** [comment]
I agree. The broken link in the Readme and the _wrong_ link in the repo description are quite misleading.
